zipline/algorithm.py

```diff
@@ -1218,6 +1218,10 @@ def order(self,
             The amount of shares to order. If ``amount`` is positive, this is
             the number of shares to buy or cover. If ``amount`` is negative,
             this is the number of shares to sell or short.
+        limit_price : float, optional
+            The limit price for the order.
```

Comment:
We might want to say somewhere that passing `limit_price=N` is equivalent to (and mutually-exclusive with) passing `style=LimitOrder(N)`, with the same going for `stop_price` with `StopOrder(N)`, and with `StopLimitOrder` when both are passed.